benchmark/vochain_benchmark_csp_test.go

```diff
@@ -0,0 +1,145 @@
+package test
+
+import (
+	"encoding/json"
+	"fmt"
+	"sync/atomic"
+	"testing"
+
+	"go.vocdoni.io/dvote/api"
+	"go.vocdoni.io/dvote/client"
+	"go.vocdoni.io/dvote/crypto/ethereum"
+	"go.vocdoni.io/dvote/log"
+	"go.vocdoni.io/dvote/test/testcommon"
+	"go.vocdoni.io/dvote/test/testcommon/testutil"
+	"go.vocdoni.io/dvote/types"
+	"go.vocdoni.io/dvote/util"
+	"go.vocdoni.io/dvote/vochain"
+	models "go.vocdoni.io/proto/build/go/models"
+	"google.golang.org/protobuf/proto"
+)
+
+func BenchmarkVochainCSP(b *testing.B) {
+	log.Init("info", "stdout")
+	b.ReportAllocs()
+	var dvoteServer testcommon.DvoteAPIServer
+	host := *hostFlag
+	if host == "" {
+		dvoteServer.Start(b, "file", "census", "vote")
+		host = dvoteServer.ListenAddr
+	}
+
+	cl, err := client.New(host)
+	if err != nil {
+		b.Fatal(err)
+	}
+
+	// create random key batch
+	keySet := testcommon.CreateEthRandomKeysBatch(b, b.N)
+	log.Infof("generated %d keys", len(keySet))
+
+	// setup the basic parties
+	signerPubHex, _ := dvoteServer.Signer.HexString()
+	signerPub := testutil.Hex2byte(b, signerPubHex)
+	cspKey := ethereum.NewSignKeys()
+	if err := cspKey.Generate(); err != nil {
+		b.Fatal(err)
+	}
+	entityID := signerPub[:types.EntityIDsize]
```

Comment:
`entityID := dvoteServer.Signer.Address().Bytes()`

Reply:
With this you do not need to import `"go.vocdoni.io/dvote/test/testcommon/testutil"` & `"go.vocdoni.io/dvote/types"`